packages/babel-plugin-strip-runtime/src/utils/preserve-leading-comments.ts

```diff
@@ -0,0 +1,18 @@
+import type { NodePath } from '@babel/traverse';
```

Comment:
This looks familiar! :)

If it is the same as [packages/babel-plugin/src/utils/preserve-leading-comments.tsx](https://github.com/atlassian-labs/compiled/blob/master/packages/babel-plugin/src/utils/preserve-leading-comments.tsx) , can we place it under `packages/utils/src`?

Reply:
Ah nice one! Didn't see that folder before - I didn't want to introduce a dependency between the two babel plugins which is why I duplicated it. Will move it to the utils folder 👍

Reply:
I have done this in a separate PR so I can bump the @compiled/utils package here https://github.com/atlassian-labs/compiled/pull/1109